### PR TITLE
Task2

### PR DESCRIPTION
**Pull request by 01yukti**

### CODE REVIEW

File: level1/alb.tf [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
```
Comment (vladimirmukhin) [inline level1/alb.tf]: ssh is not needed for load balancer

File: level1/alb.tf [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+#configure security group
+resource "aws_security_group" "alb_sg" {
+  name   = "alb_sg"
+  vpc_id = aws_vpc.example.id
+
+  ingress {
+    from_port   = 80
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
```
Comment (vladimirmukhin) [inline level1/alb.tf:66]: It's better to close it down to the ISP range as well

File: level1/alb.tf [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    to_port          = 0
+    protocol         = "-1"
+    cidr_blocks      = ["0.0.0.0/0"]
+    ipv6_cidr_blocks = ["::/0"]
+  }
+
+  tags = {
+    Name = "alb_sg"
+  }
+}
```
Comment (vladimirmukhin) [inline level1/alb.tf]: Trailing newline is missing

**Review by vladimirmukhin — CHANGES_REQUESTED** [review]

File: level1/alb.tf (hunk 1 above)
Comment (01yukti) [reply level1/alb.tf]: ok understood

File: level1/alb.tf (hunk 2 above)
Comment (01yukti) [reply level1/alb.tf:66]: I dont understand this one..what is ISP range?

File: level1/alb.tf (hunk 3 above)
Comment (01yukti) [reply level1/alb.tf]: added

**Pull request merged**